.travis.yml

```diff
@@ -4,14 +4,16 @@ env:
 language: go
 go:
   - 1.12.x
+  - 1.8.x
   - master
 before_install:
+  - go get github.com/go-playground/overalls
   - go get github.com/mattn/goveralls
 git:
   depth: 1
 script:
-  - go test -v -race ./... -coverprofile=profile.cov
```

Comment:
yes because `overalls` does this for all versions of go, the previous approach was not supported by go 1.8.x.

Reply:
@juancarlostong could there be a better to run this where instead of ignoring we whitelist which packages we want to test? What are y'all doing in Sidedoor?